metals/src/main/scala/scala/meta/internal/decorations/SyntheticsDecorationProvider.scala

```diff
@@ -297,13 +299,12 @@ final class SyntheticsDecorationProvider(
         val edit = buffer.tokenEditDistance(path, textDocument.text)
         val decorations = for {
           synthetic <- textDocument.synthetics
-          range <- synthetic.range.toIterable
-          decoration <- printSyntheticInfo(
+          (decoration, range) <- printSyntheticInfo(
             textDocument,
             synthetic,
             toDecorationString(textDocument),
             userConfig(),
-            simple = true
+            isHover = false
           ).toIterable
```

Comment:
I know you didn't change this, but `toIterable` isn't actually needed here right?

Reply:
I will go through and remove any that are not needed anymore. Good catch!

Reply:
Removed 2 `toIterable` altogether.